rlenvs/Env.lua

```diff
@@ -3,16 +3,27 @@ local classic = require 'classic'
 local Env = classic.class('Env')
 
 -- Denote interfaces
-Env:mustHave('start')
+Env:mustHave('_start')
 Env:mustHave('_step')
 Env:mustHave('getStateSpace')
 Env:mustHave('getActionSpace')
 Env:mustHave('getRewardSpace')
 
+function Env:_init(opts)
+    if opts.timeStepLimit and opts.maxSteps then
+        self.maxSteps = math.min(opts.timeStepLimit, opts.maxSteps)
```

Comment:
Is this how `gym` does things? To me it makes more sense that if the user specifies `maxSteps`, then this should override any defaults. And then if `timeStepLimit` is available, that should be used. Then `1000`. Makes the logic a little bit simpler too.